R/applyScale.R

```diff
@@ -17,7 +17,7 @@
 #' chanIdx <- 5
 #' applyScale(scaleSet$scales[[1]][chanIdx, "scale"], c(1, 2, 3, 4, 5))
 #' }
-applyScale <- function(scale, data, clamp_q = FALSE) {
```

Comment:
Need to document this as breaking in the changelog.

Reply:
Ah, yes, whoops; would it be better if I pulled this commit into a different branch for a PR into `v1-dev`?

Reply:
Yeah, or support both for now.

Reply:
If yes, it's [here](https://github.com/primitybio/cellengine-r-toolkit/pull/71), and I'll drop this commit in favor of:
```r
# .lintr
object_name_linter = object_name_linter(c("camelCase", "snake_case")),
```

And then change that to 
# .lintr
object_name_linter = object_name_linter("camelCase"),
```
in #71